source/guide.md

```diff
@@ -0,0 +1,441 @@
+
+[Preprocessing](#topic-1)
+[Variables](#topic-2)
+[Nesting](#topic-3)
+[Partials](#topic-4)
+[Modules](#topic-5)
+[Mixins](#topic-6)
+[Inheritance](#topic-7)
+[Operators](#topic-8)
+[Lists](#topic-9)
+
+</nav> -->
+
+## Preprocessing
+
+CSS on its own can be fun, but stylesheets are getting larger, more
+complex, and harder to maintain. This is where a preprocessor can help.
+Sass has features that don't exist in CSS yet like nesting, mixins,
+inheritance, and other nifty goodies that help you write robust,
+maintainable CSS.
+
+Once you start tinkering with Sass, it will take your preprocessed Sass
+file and save it as a normal CSS file that you can use in your website.
+
+The most direct way to make this happen is in your terminal. Once Sass is
+installed, you can compile your Sass to CSS using the `sass` command.
+You'll need to tell Sass which file to build from, and where to output CSS
+to. For example, running `sass input.scss output.css` from your terminal
+would take a single Sass file, `input.scss`, and compile that file to
+`output.css`.
+
+You can also watch individual files or directories with the `--watch`
+flag. The watch flag tells Sass to watch your source files for changes,
+and re-compile CSS each time you save your Sass. If you wanted to watch
+(instead of manually build) your `input.scss` file, you'd just add the
+watch flag to your command, like so:
+
+```
+sass --watch input.scss output.css
+```
+
+You can watch and output to directories by using folder paths as your
+input and output, and separating them with a colon. In this example:
+
+{% renderFile 'source/code-snippets/_homepage-sass-watch.md' %}
+
+Sass would watch all files in the `app/sass` folder for changes, and
+compile CSS to the `public/stylesheets` folder.
+
+<div class="sl-c-callout sl-c-callout--fun-fact">
+ 
+### 💡 Fun fact:
+
+Sass has two syntaxes! The SCSS syntax (`.scss`) is used most commonly. It's
+a superset of CSS, which means all valid CSS is also valid SCSS. The
+indented syntax (`.sass`) is more unusual: it uses indentation rather than
+curly braces to nest statements, and newlines instead of semicolons to
+separate them. All our examples are available in both syntaxes.
+
+</div>
+
+---
+
+## Variables
+
+Think of variables as a way to store information that you want to reuse
+throughout your stylesheet. You can store things like colors, font stacks,
+or any CSS value you think you'll want to reuse. Sass uses the `$` symbol
+to make something a variable. Here's an example:
+
+{% assign code = "$font-stack: Helvetica, sans-serif;
+$primary-color: #333;
+
+body {
+font: 100% $font-stack;
+color: $primary-color;
+}
+
+===
+
+$font-stack: Helvetica, sans-serif
+$primary-color: #333
+
+body
+font: 100% $font-stack
+color: $primary-color" | codeExample: true, 'scss' -%}
+
+{% assign example_id = 1 %}
+{% assign ui_id = 1 %}
+{% render 'code_example',
+code: code,
+example_id: example_id,
+ui_id: ui_id %}
```

Comment:
Hrm, this is tricky... The liquid file is included here in a Markdown template, where it is then parsed as Markdown. So that means whitespace in the Liquid template is significant, which is causing extra `<p>` tags where we don't want them (specifically when the code sample contains two consecutive newlines `\n\n`). 😖 

I'm struggling to think of a good solution... I think it's a fundamental problem/risk any place we're embedding markup in a markdown file. Maybe our standard approach should shift to Liquid templates, with Markdown embedded selectively, instead of the other way around? Markdown is dangerous when it's used in unexpected places! 😅